agent/src/main/java/com/code_intelligence/jazzer/api/Severity.java

```diff
@@ -0,0 +1,39 @@
+// Copyright 2021 Code Intelligence GmbH
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.code_intelligence.jazzer.api;
+
+/**
+ * The severity level of a detected security issue.
+ */
+public enum Severity {
+  /**
+   * Examples of security issues of critical severity include: remote code execution
+   */
+  CRITICAL,
+  /**
+   * Examples of security issues of high severity include: SQL injections
```

Comment:
These could use some more examples (as well as consensus on whether these rankings are indeed sensible). @sebastianpoeplau Do you have some ideas on what kind of issues to add here that apply to memory-safe languages?

Reply:
We could take some items from the OWASP Top 10 list, no? E.g. "Broken Authentication", "Sensitive Data Exposure" could be checked with specialized fuzz targets.

Reply:
I think that the severity of such rather generic issues would depend a lot more on context than for something like RCE. I have weakened the wording a bit.